### PR TITLE
#97 More Forms UX improvements

### PR DESCRIPTION
**Pull request by jasonadt**
## Description

  * Fixed more forms style/UX issues (reported by design, see issue here: https://github.com/lana/b2c-mapp-ui/issues/97 for more details)
  * Updated NPM dependencies
  * Bumped the `PATCH` version in `package.json

## Testing

  * `npm run lint`: **PASS**
  * `npm run test`: **PASS**
  * `npm run prepare`: **PASS**
  * Local testing with Storybook: **PASS**

## Checks
- [ ] Requires documentation update
- [x] Requires lib version update

## Versioning impact
- [ ] **MAJOR** incompatible API changes.
- [ ] **MINOR** adds functionality in a backwards-compatible manner.
- [x] **PATCH** backwards-compatible bug fixes.

### CODE REVIEW

**Review by rahulpsingh — APPROVED** [review]

**Review by TanMar23 — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -133,5 +133,6 @@ module.exports = {
     }],
     'vue/v-on-function-call': 'error',
     'vue/valid-v-slot': 'error',
+    'vue/custom-event-name-casing': 'off',
```
Comment (jantocv) [inline .eslintrc.js:136]: 👌

**Review by jantocv — APPROVED** [review]
LGTM 🚀

**Comment by jantocv** [comment]
should we consider a unit-test case to cover that improvement? @jasonadt

File: src/components/CurrencyField/CurrencyField.scss [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 .field-container {
   background: $white;
-  border: 1px solid $black-300;
+  border: 1.5px solid $black-300;
```
Comment (JoseOrtiz) [inline src/components/CurrencyField/CurrencyField.scss:6]: I think fractional pixels are truncated

Comment (jasonadt) [reply src/components/CurrencyField/CurrencyField.scss:6]: I thought so too, but it actually does work and it makes a difference (this value was specifically requested by the design team)

**Comment by jasonadt** [comment]
> should we consider a unit-test case to cover that improvement? @jasonadt

As discussed over Slack: I don't think that it would be needed since this change does not affect any user-interaction (it's pure styling) and the transition is very quick so it'd be painful to test properly (with little to no value in the end). Thanks for asking though!

**Pull request merged**